packages/vsx-registry/src/browser/vsx-extension.tsx

```diff
@@ -527,7 +537,7 @@ export class VSXExtensionEditorComponent extends AbstractVSXExtensionComponent {
             </div>
             {
                 sanitizedReadme &&
-                < div className='scroll-container'
+                <div className='scroll-container'
```

Comment:
Is this an error caused by the previous formatting?  I think there should be no spaces here.